src/dll/Utils.cpp

```diff
@@ -230,3 +231,8 @@ std::wstring Utils::Widen(const std::string_view aText)
 
     return result;
 }
+
+void Utils::ToLower(std::wstring& aText)
```

Comment:
```suggestion
std::wstring Utils::ToLower(const std::wstring& aText)
```

I prefer to have it as a return parameter since it makes it more readable at first glance.

Reply:
In this case, implementation use `std::transform` which change `aText` in-place. Hence the use of a reference only.
I don't think it is better to pass `aText` with `const &`, it would require to create a copy within method. What do you think?

Reply:
If you prefer `const &` with return type anyway. Let me know.

Reply:
> In this case, implementation use `std::transform` which change `aText` in-place. Hence the use of a reference only. I don't think it is better to pass `aText` with `const &`, it would require to create a copy within method. What do you think?

I prefer to have a copy that is transformed and then returned. Imo using the in param as out is also a bit of a code smell.